src/__tests__/starWarsQuery-test.js

```diff
@@ -170,71 +139,65 @@ describe('Star Wars Query Tests', () => {
 
   describe('Using IDs and query parameters to refetch objects', () => {
     it('Allows us to query for Luke Skywalker directly, using his ID', async () => {
-      const query = `
-        query FetchLukeQuery {
+      const result = await executeQuery(`
```

Comment:
These are intended to be full integration tests, so they intentionally were running the full graphql function instead of just the execute step

Reply:
@leebyron it's a naming issue `executeQuery` is defined like this:
```js
function executeQuery(query, variableValues) {
  return graphql(StarWarsSchema, query, new Query(), null, variableValues);
}
```
Since now in majority of tests you need to supply 3 args instead of 2:
```js
const result = graphql(StarWarsSchema, query, new Query());
```
I decided to move it into utility function but now I see that it needs a better name. Do you have any suggestions?